config/applets/color_mesh.conf

```diff
@@ -13,11 +14,14 @@ all_frames = false
 
 # We compare the depth buffer value with the depth of the mesh point.
 # We use threshold >= 0 to fix floating point inaccuracies
-# Default value is 0, bigger values will remove more points.
-occlusion_threshold = 0.0
+# Default value is 1, bigger values will remove more points.
+occlusion_threshold = 1
 
-# Remove occluded points if parameter is true.
+# Do not color occluded points if parameter is true.
 remove_occluded = true
```

Comment:
I wonder if this should now be called `color_occluded` and have the opposite meaning as it currently does.  That is
`remove_occluded = true` becomes `color_occluded = false`

Reply:
Yes, I was thinking exactly that.